### ISSUE TITLE
Some more useful papers

### ISSUE THREAD

**Issue by shantanu-ai** [aisagarw/awesome-explainable-cv#1]
Hi @aisagarw,

I want to contribute with these papers. Did not have the write access to create a branch and submit a pull request. So creating this issue, can u please add the following papers to this list.

- **"Logic Explained Networks"**  
    **Authors**: _Gabriele Ciravegna, Pietro Barbiero, Francesco Giannini, Marco Gori,
Pietro Liò, Marco Maggini, Stefano Melacci_ <br>
    **Journal**: [Artificial Intelligence](https://arxiv.org/abs/2108.05149)  <br>
    **Summary**: _Logic Explained Networks (LENs) offer interpretable deep learning models that provide human-understandable explanations using First-Order Logic, outperforming traditional white-box models in both supervised and unsupervised learning tasks._

- **"Entropy-based Logic Explanations of Neural Networks"**  
    **Authors**: _Pietro Barbiero, Gabriele Ciravegna, Francesco Giannini, Pietro Lió, Marco Gori, Stefano Melacci_ <br>
    **Conference**: [AAAI 2022](https://arxiv.org/abs/2106.06804)  <br>
    **Summary**: _A novel end-to-end approach extracts concise First-Order Logic explanations from neural networks using an entropy-based criterion, improving both interpretability and classification accuracy in safety-critical domains._

- **"Dividing and Conquering a BlackBox to a Mixture of Interpretable Models: Route, Interpret, Repeat"**  
    **Authors**: _Shantanu Ghosh, Ke Yu, Forough Arabshahi, Kayhan Batmanghelich_ <br>
    **Conference**: [ICML 2023](https://proceedings.mlr.press/v202/ghosh23c.html)  <br>
    **Summary**: _This paper introduces a method to iteratively carve concept-based interpretable models from a Blackbox in a post-hoc manner, using First Order Logic for explanations, while a residual network handles harder cases, achieving high interpretability without sacrificing performance._

- **"Distilling BlackBox to Interpretable models for Efficient Transfer Learning"**  
    **Authors**: _Shantanu Ghosh, Ke Yu, Forough Arabshahi, Kayhan Batmanghelich_ <br>
    **Conference**: [MICCAI 2023](https://link.springer.com/epdf/10.1007/978-3-031-43895-0_59?sharing_token=vDZQRGbOgg_uNMjDCTkmSPe4RwlQNchNByi7wbcMAY7LayIvXqtIKyotmDxwKVoR3vE6_tVF4ADNzhgWf_E8lqdZHGLlVhLQsT-f7FM21A5VImVHfgQYdYURRq1z5yRUoC2j2qsFEWUXudYxHYcBdpzFCUEMAvrfxYUsk5bkEdM%3D)  <br>
    **Summary**: _This paper presents an concept-based interpretable model for chest-X-ray classification that can be efficiently fine-tuned for new domains using minimal labeled data, leveraging semi-supervised learning and distillation from blackbox models._

Regards 
Shantanu

**Comment by shantanu-ai** [comment]
Sorry i just submitted the PR.
